main.go

```diff
@@ -41,7 +40,6 @@ var (
 func init() {
 	setting.AppVer = Version
 	setting.AppBuiltWith = formatBuiltWith()
-	setting.AppStartTime = time.Now().UTC()
```

Comment:
This change seems unrelated, could you describe why this is being removed?

Reply:
I's becoming unused because the `/manifest.json` route was the only consumer. We can keep it around if you prefer.

Reply:
Hmm, I think it's duplicate with this line also.
https://github.com/go-gitea/gitea/blob/4aabbacdfe2830e3b3808e42d6d89d6688754089/routers/admin/admin.go#L43

Reply:
I've restored `setting.AppStartTime` and it's now being used on the admin page:

<img width="459" alt="image" src="https://user-images.githubusercontent.com/115237/102831540-59086f80-43ec-11eb-9ec1-4ed5bea82611.png">